jsoniter-scala-benchmark/jvm/src/test/scala/com/github/plokhotnyuk/jsoniter_scala/benchmark/ADTWritingSpec.scala

```diff
@@ -14,6 +14,7 @@ class ADTWritingSpec extends BenchmarkSpecBase {
       toString(b.jsoniterScala()) shouldBe b.jsonString1
       toString(b.preallocatedBuf, 0, b.jsoniterScalaPrealloc()) shouldBe b.jsonString1
       toString(b.ninnyJson()) shouldBe b.jsonString3
+      toString(b.ninnyJsonJsoniter()) shouldBe b.jsonString3 // todo: how do we get this to write compact BigDecimals?
```

Comment:
jsoniter is writing big decimals with an empty fraction (eg. `2.0`) is there a way to avoid this?

Reply:
Probably checking the scale and serializing it as `BigInt` would work.